src/main/scala/com/metl/model/BackendAdaptor.scala

```diff
@@ -455,6 +455,7 @@ class ManagedCache[A <: Object,B <: Object](name:String,creationFunc:A=>B,cacheC
     .memoryStoreEvictionPolicy(cacheConfig.memoryEvictionPolicy)
     .persistence(new PersistenceConfiguration().strategy(PersistenceConfiguration.Strategy.NONE))
     .logging(false)
+  cacheConfig.timeToLiveSeconds.foreach(s => cacheConfiguration.timeToLiveSeconds(s))
```

Comment:
It seems like not (but no I haven't tested by dropping the expiry to really short):

/**
     * Builder which sets the time to idle for an element before it expires. Is only used if the element is not eternal.
     * This default can be overridden in {@link net.sf.ehcache.Element}
     * <p/>
     * This property can be modified dynamically while the cache is operating.
     *
     * @param timeToLiveSeconds the default amount of time to live for an element from its creation date
     * @return this configuration instance
     * @see #setTimeToLiveSeconds(long)
     */
    public final CacheConfiguration timeToLiveSeconds(long timeToLiveSeconds) {
        setTimeToLiveSeconds(timeToLiveSeconds);
        return this;
    }